external/@wordpress/blocks/index.js

```diff
@@ -19,14 +28,20 @@ export function registerBlock( namespace, block, settings ) {
  * @return {?Object}           Block settings
  */
 export function getBlockSettings( namespace, block ) {
+	if ( ! blocks[ namespace ] || ! blocks[ namespace ][ block ] ) {
+		return null;
+	}
 
+	return blocks[ namespace ][ block ];
 }
 
 /**
  * Returns all registered blocks.
  *
- * @return {Object} Block settings keyed by block name
+ * @return {Array} Blocks settings
```

Comment:
What are the expected uses of this function?  Do you think we will need the ability to pull a specific block out of the list?  If so, it might be a good idea to return an array keyed by namespace then block name, as the blocks are stored.

Also, `Block settings` is better wording.